pages/apim/3.x/user-guide/publisher/plans/plans-and-subscriptions.adoc

```diff
@@ -0,0 +1,55 @@
+= Plans and subscriptions
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_plans_subscriptions.html
+:page-folder: apim/user-guide/publisher
+:page-keywords: Gravitee.io, API Platform, API Management, API Gateway, documentation, manual, guide, reference, api, CGU, GCU
+:page-layout: apim3x
+
+== Overview
+
+Once an API is registered and made public, you can manage subscriptions to it through APIM plans. Managing subscriptions and plans is a key feature of APIM that publishers can use to provide and regulate access to APIs.
+
+== What is a plan?
+
+A plan provides a service and access layer on top of your APIs for consumer link:/apim/3.x/apim_overview_concepts.html#gravitee-concepts-application[applications].
+A plan specifies access limits, subscription validation modes and other configuration to tailor it to a specific application.
+
+image::apim/3.x/api-publisher-guide/plans-subscriptions/plan-diagram.png[Gravitee.io - Plan diagram, 873, 530, align=center, title-align=center]
```

Comment:
not sure about this image. In the case of keyless plan, there is nor application neither subscription. It's the case only for secured plans.